test/e2e/migration/migration.go

```diff
@@ -167,8 +167,8 @@ func MigrationTest(useVolumeSnapshots bool, veleroCLI2Version VeleroCLI2Version)
 					OriginVeleroCfg.Plugins = ""
 					//TODO: Remove this setting when migration path is from 1.13 to higher version
 					//TODO: or self, because version 1.12 and older versions have no this parameter.
-					OriginVeleroCfg.WithoutDisableInformerCacheParam = true
-					OriginVeleroCfg.DisableInformerCache = false
+					OriginVeleroCfg.WithoutEnableInformerCacheParam = true
+					OriginVeleroCfg.EnableInformerCache = true
```

Comment:
Please help to remove line 171: _OriginVeleroCfg.EnableInformerCache = true_, it's not used since .WithoutEnableInformerCacheParam = true